include/hip/hip_runtime_api.h

```diff
@@ -136,7 +136,7 @@ typedef struct hipDeviceProp_t {
                                                     ///devices with unmatched block dimensions
     int cooperativeMultiDeviceUnmatchedSharedMem;   ///< HIP device supports cooperative launch on multiple
                                                     ///devices with unmatched shared memories
-
+    int isLargeBar;                  ///< 1: if it is a large PCI bar device, else 0
```

Comment:
Wondering why it can't be 'bool' when only two valid values allowed.

Reply:
I followed the examples in this same struct. They use int to express bool.

Reply:
Hi,
I appreciate all your comments. I will close the request now. I may address your comment in future.
Thanks,
Alex